core/persist/viewmanager/ViewManagerModel.ts

```diff
@@ -272,55 +251,49 @@ export class ViewManagerModel<T extends PlainObject = PlainObject>
     }
 
     override async doLoadAsync(loadSpec: LoadSpec) {
-        const rawViews = await XH.jsonBlobService.listAsync({
-            type: this.viewType,
-            includeValue: true,
-            loadSpec
-        });
+        const rawViews = await XH.jsonBlobService.listAsync({type: this.viewType, loadSpec});
```

Comment:
Makes sense you removed `includeValue` because it defaults to `true` in `JsonBlob.formatForClient`. I think that default is a little hard to find. Worth a comment in `JsonBlobService` on the client and/or `XhController.listJsonBlobs`?

Reply:
We resolved this -- default is actually not to include them!  Clarified in code, with explicit line however